bot/exts/evergreen/snakes/_snakes_cog.py

```diff
@@ -1126,26 +1126,15 @@ async def zen_command(self, ctx: Context) -> None:
     # endregion
 
     # region: Error handlers
-    @get_command.error
     @card_command.error
-    @video_command.error
     async def command_error(self, ctx: Context, error: CommandError) -> None:
         """Local error handler for the Snake Cog."""
-        embed = Embed()
-        embed.colour = Colour.red()
-
-        if isinstance(error, BadArgument):
-            embed.description = str(error)
-            embed.title = random.choice(ERROR_REPLIES)
-
-        elif isinstance(error, OSError):
-            log.error(f"snake_card encountered an OSError: {error} ({error.original})")
+        original_error = error.original
```

Comment:
Not all errors will have this attribute, so use `getattr` or `hasattr`.

Reply:
alright